tests/Integration/TestCase.php

```diff
@@ -48,5 +48,10 @@ protected function queueListen(?string $queue = null): void
         $process = new Process($command);
         $this->processes[] = $process;
         $process->start();
+
+        usleep(500000);
```

Comment:
It's not really good to sleep during the tests

Reply:
Agreed. But we should ensure that the daemon is working. Do you have any ideas on how to achieve this goal without using usleep?

Reply:
Let's discuss it in an issue, if you think this should be changed.